pkg/cmd/uninstall/uninstall.go

```diff
@@ -60,6 +60,13 @@ func NewCmdUninstall(commonOpts *opts.CommonOptions) *cobra.Command {
 }
 
 func (o *UninstallOptions) Run() error {
+	fmt.Println("")
+	uninstall := util.Confirm("Uninstall JX - this command will remove all JX components and delete all namespaces created by Jenkins X. Do you wish to continue?", false,
```

Comment:
It might be nice to present the user with a list of what will be deleted here...

Reply:
Yeah I can do that

Reply:
Sorry, I actually meant the names of the namespaces that will be deleted...

Reply:
ah ok, misunderstood. I could do that instead.